### PR TITLE
Scripts/Outland: Migrate Shattrath City NPCs to SAI

### PR DESCRIPTION
**Pull request by mysterious-static**
**Changes proposed:**

-  Migrates NPC's in Shattrath City to SAI.

**Target branch(es):** 
- [X] 3.3.5

**Tests performed:** (Does it build, tested in-game, etc.)
Builds, tested in-game and behaves as expected.

**Known issues and TODO list:** (add/remove lines as needed)

- [x] Verify gossip menu 8751, 8752 option ID 0 text on live - does anyone have a character exalted with the proper factions to check on this? I do not, or I would have done it.
- [x] Test ingame to make sure all the conditions work.
- [x] **Reviewers**: Is the first TextID for a gossip menu always the default? If so, the NegativeConditions can be removed from the condition insert.
- [x] Refactor sql, I'm leaving it the way it is right now so it's readable for reviewers.

### CODE REVIEW

**Comment by tkrokli** [comment]
Feel free to remove the [WIP] part of the PR title when you have made the necessary changes (according to your  TODO list). 👍

**Comment by velinath** [comment]
pr is ready for review; could still use verification on live if anyone has a character with which they can test for the gossip menu option for the reputation vendors

**Comment by velinath** [comment]
I still don't have access to a character with the proper prerequisites to check the gossip menu option text, so unfortunately this is as much as I can do with it. I'll consolidate the SQL shortly

File: sql/updates/world/3.3.5/9999_99_99_99_world.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+--
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` IN (25967,23483,23484);
+
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
```
Comment (Killyana) [inline sql/updates/world/3.3.5/9999_99_99_99_world.sql]: Delete statement here.

File: sql/updates/world/3.3.5/9999_99_99_99_world.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+(8752,11085,0);
+
```
Comment (Killyana) [inline sql/updates/world/3.3.5/9999_99_99_99_world.sql]: And here.

**Comment by tkrokli** [comment]
Just to be absolutely clear: You don't need a sniff of the gossip, you need a **screenshot** or **video** from **retail** to verify what the actual gossip option is for a player who is **exalted** with:
- http://www.wowhead.com/faction=942/cenarion-expedition
- http://www.wowhead.com/faction=935/the-shatar
  - and one of the 2 main factions :
- http://www.wowhead.com/faction=932/the-aldor
- http://www.wowhead.com/faction=934/the-scryers

File: sql/updates/world/3.3.5/9999_99_99_99_world.sql (hunk 1 above)
Comment (tkrokli) [reply sql/updates/world/3.3.5/9999_99_99_99_world.sql]: Ah, right. Add the missing `DELETE FROM` line above the `INSERT INTO` line (row).

**Comment by velinath** [comment]
nah you've got it, that's what would be needed

sniff'd be fine too, honestly, but realistically I just need to know what the gossip option says

**Comment by Killyana** [comment]
Here:
``` sql
UPDATE `npc_text` SET `BroadcastTextID0`=21912 WHERE `id`=11083;
UPDATE `npc_text` SET `BroadcastTextID0`=21913 WHERE `id`=11084;
UPDATE `npc_text` SET `BroadcastTextID0`=21914 WHERE `id`=11085;

DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (8751,8752);
INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
(8751, 0, 1, 'I have marks to redeem.', 21915, 3, 128, 0, 0, 0, 0, '', 0, 0),
(8752, 0, 1, 'I have marks to redeem.', 21915, 3, 128, 0, 0, 0, 0, '', 0, 0);

**Comment by velinath** [comment]
Thanks @Killyana, I'll get this in there momentarily. Appreciate your looking into this one, I wish I could have done it myself but c'est la vie I guess.

**Review by ghost — APPROVED** [review]
Looks good to me. ™

**Comment by tkrokli** [comment]
Any reasons not to merge this now (and rename the SQL file `2017_10_16_02_world.sql` in the process)?

**Pull request merged**